org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/util/configuration/FixedRequestAttributesBuilder.java

```diff
@@ -79,6 +79,9 @@ public static FixedRequestAttributesBuilder fromWorkspaceSettings(File projectDi
         if (configuration.isOffline()) {
             result.arguments.add("--offline");
         }
+        if (configuration.isBuildScansEnabled()) {
+            result.jvmArguments.add("-Dscan");
```

Comment:
Starting with Gradle 3.4 and scan plugin 1.6 a deprecation warning is shown, saying "Use --scan instead". We need to figure out how to work around this in the mid term. Let's just open a card for now.